ts/output/common/Wrappers/mo.ts

```diff
@@ -357,6 +367,22 @@ export function CommonMoMixin<T extends WrapperConstructor>(Base: T): MoConstruc
       return [h - d, d];
     }
 
+    /**
+     * Determine the size of the delimiter based on whether full extenders should be used or not.
+     *
+     * @param {number} D          The requested size of the delimiter
+     * @param {DelimiterData} C   The data for the delimiter
+     * @return {number}           The final size of the assembly
+     */
```

Comment:
This should just be an `@override` jsdoc.

Reply:
I've never fully understood when to use `@override`.  You seem to be saying that if it is described in the interface, then the code that implements that interface should be marked `@override`.  In that case, most of the jsdoc in this class (and in lost of other places) should be turned into `@override`.  Perhaps another cleanup PR is in order?